### PR TITLE
Laboratory work #3, Eliseykina Ekaterina - 20FPL1

### PR DESCRIPTION
**Pull request by reginafeles**

### CODE REVIEW

**Comment by WhiteJaeger** [comment]
It is time to push some changes for the third lab.

**Comment by WhiteJaeger** [comment]
It is time to push some changes for the 3rd lab.

**Comment by reginafeles** [comment]
I do not understand why my code fails tests there. Everything is good in PyCharm. Maybe it happens because of lab_2? Should I del these files?

File: lab_3/main.py [hunk 1]
```diff
@@ -20,16 +23,63 @@ def tokenize_by_sentence(text: str) -> tuple:
...
+
+    for sentence in text_list:
+        ready_words = []
+        sentence = sentence.strip()
+        sentence = sentence + ' '
+        tokenized_word = ['_']
+        for word in sentence:
+            if word != ' ':
+                tokenized_word.append(word)
+            if word == ' ':
```
Comment (WhiteJaeger) [inline lab_3/main.py:54]: Could be done with `elif`.

**Review by WhiteJaeger — CHANGES_REQUESTED** [review]

File: lab_3/main.py (hunk 1 above)
Comment (reginafeles) [reply lab_3/main.py]: done

File: lab_3/main.py [hunk 2]
```diff
@@ -222,7 +334,24 @@ def save(self, name: str) -> int:
...
+                for integer in n_gram:
+                    word += self.storage.get_letter_by_id(integer)
+                freq[word] = frequency
+                word = ''
+        profile_as_dict['freq'] = freq
+        profile_as_dict['n_words'] = self.n_words
+        profile_as_dict['name'] = self.language
+        with open(name, 'w', encoding='UTF-8') as file:
+            json_string = json.dumps(profile_as_dict)
+            file.write(json_string)
```
Comment (WhiteJaeger) [inline lab_3/main.py:353]: There is the `.dump` method in the `json` package that combines these two lines.

**Review by WhiteJaeger — CHANGES_REQUESTED** [review]

File: lab_3/main.py (hunk 2 above)
Comment (reginafeles) [reply lab_3/main.py]: done

**Pull request closed without merging**